lib/tpm2_options.c

```diff
@@ -338,13 +338,14 @@ tpm2_option_code tpm2_handle_options (int argc, char **argv,
         { "quiet",         no_argument,       NULL, 'Q' },
         { "version",       no_argument,       NULL, 'v' },
         { "enable-errata", no_argument,       NULL, 'Z' },
+        { "no-tpm",        no_argument,       NULL, 'X' },
```

Comment:
I'm not sure if I like the `no-tpm` option name. Maybe `use-openssl`? or `openssl-backend`?

Reply:
I agree that using `openssl-backend` is definitely more clear about what is happening than just `no-tpm` -- this long option has been updated to reflect this.  The documentation and integration test has also been updated.